supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -23,8 +23,15 @@ Follow these guidelines when linking externally:
 * Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
 * Use hyperlinks unless the URL is an example URL or is otherwise inaccessible to users. For information about using addresses in examples, see the _IBM Style_ guide.
 * By default, links are followable and crawlable. Do not use the "nofollow" link option unless absolutely necessary.
-* Avoid links to external sites when possible. External links can change or be unreliable.
 * Avoid deep links (to a specific page or image from another company) to prevent an ongoing maintenance responsibility and to prevent bypassing a website's legal information.
+* Avoid links to external sites when possible. External links can change or be unreliable.
+External sites are considered to be sites other than those owned and operated by Red Hat.
++
+[NOTE]
+====
+Links to upstream documentation sites are considered to be external links.
+Linking to an upstream site does not imply that Red Hat supports or endorses any content on that site.
```

Comment:
> Linking to an upstream site does not imply that Red Hat supports or endorses any content on that site.

This line seems out of place -- it's not really guidance about using cross-references to external links, is it?

Reply:
@sbmetz Yeah, I see what you mean. I was picking up the need for this from the Slack thread linked in Issue #291. Maybe it would be best to leave this out. I'd be interested in what others have to say.

Reply:
Agree that we should delete the second note -- if we want to keep the text about inferred endorsement, we can add it in the main bullet point as I've suggested in a comment above.

Reply:
Removed. I also made the sentence more generic to refer to all upstream sites (not just documentation) and added the example of GitHub.